### PR TITLE
vid-par

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
this pr is based on https://github.com/vesoft-inc/nebula-docs-cn/pull/68.

### CODE REVIEW

File: docs/manual-EN/5.appendix/vid-partition.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+VID mod partition_number = partition ID + 1
+```
+
+In the preceding formula,
+
+- `mod` is the modulo operation.
+- `partition_number` is the number of partition for the graph space where the `VID` is located, namely the value of `partition_num` in the [CREATE SPACE](../2.query-language/4.statement-syntax/1.data-definition-statements/create-space-syntax.md) statement.
+- `partition ID` the ID for the partition where the `VID` is located.
+
+For example, if there are 100 partitions, the vertices with `VID` 1, 11, 101, 1001 will be stored on the same partition.
```
Comment (whitewum) [inline docs/manual-EN/5.appendix/vid-partition.md]: My bad. 1, 101, 1001.  11 is not stored on the same partition.

File: docs/manual-EN/5.appendix/vid-partition.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Vertex Identifier and Partition
+
+This document provides some introductions on vertex identifier (`VID` for short) and partition.
```
Comment (oldLady344) [inline docs/manual-EN/5.appendix/vid-partition.md:3]: on --> to

File: docs/manual-EN/5.appendix/vid-partition.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+In **Nebula Graph**, vertices are identified with vertex identifiers (i.e. `VID`s). When inserting a vertex, you must specify a `VID` for it. You can generate `VID`s either with your own application or with the hash function provided by **Nebula Graph**.
+
+`VID`s must be unique in a graph space. That is, in the same graph space, vertices with the same `VID` are considered as the same vertex. `VID`s in different graph spaces are independent of each other. In addition, one `VID` can have multiple `TAG`s.
+
+When inserting data into **Nebula Graph**, vertices and edges will distribute to different partitions. And the partitions are located on different machines. If you want some certain vertices to locate on the same partition (i.e. on the same machine), you can control the generation of the `VID`s by using the following formula.
```
Comment (oldLady344) [inline docs/manual-EN/5.appendix/vid-partition.md:9]: When inserting data into **Nebula Graph**, vertices and edges will distribute to different partitions.  ==> When inserted into **Nebula Graph**, vertices and edges are distributed across different partitions.

File: docs/manual-EN/5.appendix/vid-partition.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+```text
+VID mod partition_number = partition ID + 1
+```
+
+In the preceding formula,
+
+- `mod` is the modulo operation.
+- `partition_number` is the number of partition for the graph space where the `VID` is located, namely the value of `partition_num` in the [CREATE SPACE](../2.query-language/4.statement-syntax/1.data-definition-statements/create-space-syntax.md) statement.
+- `partition ID` the ID for the partition where the `VID` is located.
```
Comment (oldLady344) [inline docs/manual-EN/5.appendix/vid-partition.md:21]: `partition ID` the ID for the partition... ==> `partition ID` is the ID for the partition...

File: docs/manual-EN/2.query-language/1.data-types/type-conversion.md [hunk 5]
```diff
@@ -8,9 +8,9 @@ Implicit conversions are automatically performed when a value is copied to a com
 
 1. Following types can implicitly converted to `bool`:
 
-- The conversions from/to bool consider `false` equivalent to `0` for empty string types, true is equivalent to all other values.
-- The conversions from/to bool consider `false` equivalent to `0` for int types, true is equivalent to all other values.
-- The conversions from/to bool consider `false` equivalent to `0.0` for float types, true is equivalent to all other values.
+   - The conversions from/to bool consider `false` equivalent to `0` for empty string types, true is equivalent to all other values.
+   - The conversions from/to bool consider `false` equivalent to `0` for int types, true is equivalent to all other values.
+   - The conversions from/to bool consider `false` equivalent to `0.0` for float types, true is equivalent to all other values.
```
Comment (oldLady344) [inline docs/manual-EN/2.query-language/1.data-types/type-conversion.md:13]: true ==> `true`?

File: docs/manual-EN/1.overview/1.concepts/1.data-model.md [hunk 6]
```diff
@@ -43,13 +43,20 @@ Edges are used to connect vertices. Each edge usually represents a relationship
...
 
-Properties are named-value pairs within vertices and edges. In our example graph, we have used the properties `id`, `name` and `age` on **player**, `id` and `name` on **team**, and `likeness` on _**like**_ edge.
+Both vertices and edges can have properties. Properties are described with key value pairs. In our example graph, we have used the properties `id`, `name` and `age` on **player**, `id` and `name` on **team**, and `likeness` on _**like**_ edge.
 
 ## Edge Rank
 
 Edge rank is an immutable user-assigned 64-bit signed integer. It affects the edge order of the same edge type between two vertices. The edge with a higher rank value comes first. When not specified, the default rank value is zero. The current sorting basis is "binary coding order", i.e. 0, 1, 2, ... 9223372036854775807, -9223372036854775808, -9223372036854775807, ..., -1.
+In addition to an edge type, the edge between two vertices must have an edge rank. The edge rank is a 64-bit integer assigned by the user; if not specified, the edge rank defaults to 0.
+
+An edge can be represented uniquely with the [source vertex, edge type, edge rank and the dest vertex].
```
Comment (oldLady344) [inline docs/manual-EN/1.overview/1.concepts/1.data-model.md:55]: [source vertex, edge type, edge rank, destination vertex].

[] indicates this is a list, so I think "and the" can be removed, and "destination vertex" is more friendly to readers.

**Pull request merged**